cypress.config.ts

```diff
@@ -5,7 +5,7 @@ export default defineConfig({
     video: false,
     screenshotOnRunFailure: false,
     pageLoadTimeout: 480000,
-    defaultCommandTimeout: 30000,
+    defaultCommandTimeout: 65000,
```

Comment:
@pepermao even after doubling the timeout, it's not working so I think there's something wrong with the test itself. Are you sure the review test spec is working fine?

Reply:
I think it might have to do with the super-admin role. When running locally it can't even create a claim, the button doesn't appear which I assume means the user is being considered regular.